cloud-config/deploy-run-scripts/app-wrapper.sh

```diff
@@ -75,7 +75,7 @@ $ENCD_CC_DIR/app-es-status.sh
 $ENCD_CC_DIR/app-pg.sh "$standby_mode"
 $ENCD_CC_DIR/app-encd.sh $ENCD_ROLE $ES_IP $ES_PORT $REGION_INDEX $APP_WORKERS
 sudo -u root $ENCD_CC_DIR/app-a2en.sh
-if [ "$ENCD_ROLE" == "demo" ]; then
+if [ "$ENCD_ROLE" == "demo" ] && [ ! "$ENCD_BUILD_TYPE" == 'encd-demo-no-es-build' ]; then
     $ENCD_CC_DIR/app-batchupgrade.sh production.ini $BUP_VAR_1 $BUP_VAR_2 $BUP_VAR_3 $BUP_VAR_4
```

Comment:
I think we said otherwise, but technically this will work with only local postgres.  It just won't match what's the in the external ES, so probably it's best to leave out.

Reply:
I think batch upgrade crashed during deployment then I added this.  I thought it was due to missing elasticsearch but I didn't look into it.